charts/postgres-operator/templates/configmap.yaml

```diff
@@ -10,9 +10,7 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/instance: {{ .Release.Name }}
 data:
-  {{- if .Values.podPriorityClassName }}
-  pod_priority_class_name: {{ .Values.podPriorityClassName }}
-  {{- end }}
+  pod_priority_class_name: {{ include "postgres-pod.priorityClassName" . }}
```

Comment:
My comment was not very clear indeed. I'll try to be clearer and more explicit.

Previously, before the changes, if `.Values.podPriorityClassName` was not set, then there would be no  `pod_priority_class_name` field in the configmap. Hence, the pod's PriorityClass is set to the default one. 
So if someone wants to set the pod's PriorityClass to the default one, they would leave the field  `.Values.podPriorityClassName` empty.

Now, with the changes, the field  `pod_priority_class_name` will always be in the configmap. It will be either `"postgres-operator.fullname"-pod` or `.Values.podPriorityClassName.name`.
Maybe, there is a case in which someone does not want to set a `pod_priority_class_name` at all to set the PriorityClass to the default one.

Reply:
ah now I see your change. Why not leave the if and change to {{- if .Values.podPriorityClassName.create }} ?

Reply:
I think it is better to put `For .Values.podPriorityClassName.create .Values.podPriorityClassName.name }}` because it covers all the cases :
1) `create: false` & `name: ""` : the pod PriorityClass is the default one
2) `create: true` & `name: ""` : the pod PriorityClass is created with the default name
3) `create: false` & `name: "my-custom-class"` : the pod PriorityClass is set to `my-custom-class`
4) `create: true` & `name: "my-custom-class"` : the pod PriorityClass is created with the name `my-custom-class`

With only  ` {{ if .Values.podPriorityClassName.create }}`, the case 3 (use of an external priority class) is not possible and it is the reason why I have open this PR

Reply:
Ok got it now. What is the default name? `default`? Can you document the behavior (what you wrote here) in the values.yaml file?

Reply:
Sure !